python/cudf/cudf/core/groupby/legacy_groupby.py

```diff
@@ -79,7 +79,17 @@ def group_min(data, segments, output):
 
 
 class Groupby(object):
-    """Groupby object returned by cudf.DataFrame.groupby().
+    """Groupby object returned by cudf.DataFrame.groupby(method="cudf").
+    `method=cudf` uses numba kernels to compute aggregations and allows
+    custom UDFs via the `apply` and `apply_grouped` methods.
+
+    Notes
```

Comment:
```suggestion
    Notes on `method='cudf'` GroupBys:
```

Reply:
Cannot change as the doctoring format allows specific sections like `Parameters`, `Notes` etc. It comes under the legacy groupby section as shown. There is `method=cudf` mentioned in a couple of other places so hopefully makes it clear.

<img width="1260" alt="Screen Shot 2019-09-24 at 12 24 47 AM" src="https://user-images.githubusercontent.com/19949207/65490841-b8cb4000-de62-11e9-8bd6-09ddc12f08a1.png">